### ISSUE TITLE
Delete Database: Warning message is not announced by the screen reader after selecting 'Delete Database' menu item.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1048]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1242733

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section.
 Turn on JAWS and Navigate to "Delete Data Base" menu item from the more options(...) of 'Sample Database' and select it.
Warning message appears along with controls of "Delete Data Base" blade.
Observe the screen Reader behavior.
Verify the issue.

Actual Result:
Warning message is not announced by the screen reader after selecting 'Delete Database' menu item.
After selecting 'Delete Database' menu item, when user navigate to the 'Delete database' blade, then screen reader does not identify the appeared warning message i.e. 'Warning! The action you are about to take cannot be undone. Continuing will permanently delete this resource and all of its children resources.

Expected Result:
Warning message should be announced by the screen reader after selecting 'Delete Database' menu item.
After selecting 'Delete Database' menu item, when user navigate to the 'Delete database' blade, then screen reader should identify and announce the appeared warning message i.e. 'Warning! The action you are about to take cannot be undone. Continuing will permanently delete this resource and all of its children resources.

Note:
Similar issue exists for error message which appears under the 'Delete' blade when user select 'ok' button without entering any data under Edit field.
This issue exists throughout the application.

https://user-images.githubusercontent.com/8519407/132716703-56a71ddb-ad53-405b-b8cd-e1d18b27f652.mp4
